conf/trainer/sparseml.yaml

```diff
@@ -0,0 +1,4 @@
+defaults:
+  - default # inherit from default trainer conf
+  - callbacks: sparseml
+  - logger: sparsewandb
```

Comment:
hey @mathemusician should we make this logger the default? seems unnecessary but let me know :)

Reply:
I answer this question in more detail below